addons/sale_project/models/project_update.py

```diff
@@ -0,0 +1,59 @@
+from odoo import api, models
+
+
+class ProjectUpdate(models.Model):
+    _inherit = 'project.update'
+
+    def _create_service_dict(self, sol, company_uom, product_uom_unit):
+        is_unit = sol.product_uom == product_uom_unit
+        product_uom_qty = sol.product_uom._compute_quantity(sol.product_uom_qty, company_uom, raise_if_failure=False)
+        qty_delivered = sol.product_uom._compute_quantity(sol.qty_delivered, company_uom, raise_if_failure=False)
+        qty_invoiced = sol.product_uom._compute_quantity(sol.qty_invoiced, company_uom, raise_if_failure=False)
+        unit = sol.product_uom if is_unit else company_uom
+
+        return {
+            'name': sol.with_context(with_price_unit=True).display_name,
+            'sold_value': product_uom_qty,
+            'effective_value': qty_delivered,
+            'invoiced_value': qty_invoiced,
+            'unit': unit.name,
+            'is_unit': is_unit,
+            'sol': sol,
+        }
+
+    def _get_common_services_values(self, project, company_uom):
+        product_uom_unit = self.env.ref('uom.product_uom_unit')
```

Comment:
product_uom_unit = self.env.ref('uom.product_uom_unit')  and company_uom (line55)

both are same